bench/bench_d.yaml

```diff
@@ -57,6 +57,11 @@ problems:
   - name: json-serde
     source:
       - 2.d
+  - name: http-server
+    source:
+      - 1.d
+      - 2.d
```

Comment:
`2.d` does not seem to exist

Reply:
Sorry, yeah. It was 3.d - I moved it to 2.d now. And remove 3.d from bench_d.yaml.